README.md

```diff
@@ -210,10 +210,11 @@ Also don't forget to checkout our [wiki](https://github.com/omarcostahamido/PVM/
 ## Development
 
 ### Logs
+The PVM execution logs will be recorded in each RPI device memory. These will be stored in the `/log/` directory, which will be created if it doesn't exist yet.
 
-All logs for each time each RPI is stored in the `log` folder in the current directory. If it doesn't exist yet, a new directory named `log` would be created.
+`pvm.py` would generate logs during execution, which will be stored in the `/log/` directory under the working directory of PVM. It would be created if it doesn't exist yet.
```

Comment:
this phrase now here sounds like a repetition of the previous one, no?

Reply:
I was trying to avoid using terms like _current directory_ or _working directory_, and thought _"the working directory of PVM."_ might be clearer. Do you have any good ideas?

Reply:
```markdown
The PVM execution logs will be recorded in each RPI device memory. These will be stored in the `/log/` directory, which will be created if it doesn't exist yet.
`pvm.py` would generate logs during execution, which will be stored in the `/log/` directory under the working directory of PVM. It would be created if it doesn't exist yet.
```
these 2 phrases seem to saying pretty much the same exact same thing. Get rid of one or try to merge?

Reply:
Sry I didn't see this till now... Done!